docker/Dockerfile-api

```diff
@@ -6,20 +6,13 @@ RUN dnf -y install \
     --setopt=install_weak_deps=false \
     --setopt=tsflags=nodocs \
     bash \
-    python3-flask \
-    python3-flask-oidc \
-    python3-gunicorn \
-    python3-ldap3 \
-    python3-neomodel \
-    python3-prometheus_client \
-    python3-six \
+    python3-pip \
     && dnf clean all
 # This will allow a non-root user to install a custom root CA at run-time
 RUN chmod 777 /etc/pki/tls/certs/ca-bundle.crt
+WORKDIR /src
```

Comment:
What is the purpose of changing the directory to /src?

Reply:
First of all, is that when I run 'pip3 install .' it would properly install estuary-api. Because when I was building this container locally it consumed all of my hard disk space in a few minutes. And second of all, is that it's more convenient to have program code in a separate place than root directory.

Reply:
The code should be placed in a folder which is not root. 
Setting up the workdir to that folder is something else. It is used for the RUN instructions context.  
Verify that this is the case.

Reply:
As I said without this instruction all the code placed into the root folder. And I saw this approach in other Dockerfiles for example in IIB, so I think it's safe to use it.